rllib/agents/dreamer/dreamer.py

```diff
@@ -77,21 +85,28 @@
         "frame_skip": 2,
     },
 
-    # Use `execution_plan` instead of `training_iteration`.
-    "_disable_execution_plan_api": False,
+    # Use `training_iteration` instead of `execution_plan`.
+    "_disable_execution_plan_api": True,
```

Comment:
Now that this is the default, let's completely remove these lines here. Less confusing for the users.

Reply:
+1

Reply:
Done!